modules/update-preparing-evaluate-apirequestcount.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
+[id="update-preparing-evaluate-apirequestcount_{context}"]
+= Using APIRequestCount to identify uses of removed APIs
+
+You can use the `APIRequestCount` API to track API requests and review whether any of them are using one of the removed APIs.
+
+.Prerequisites
+
+* You must have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Run the following command and examine the `REMOVEDINRELEASE` column of the output to identify the removed APIs that are currently in use:
++
+[source,terminal]
+----
+$ oc get apirequestcounts
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                                      REMOVEDINRELEASE   REQUESTSINCURRENTHOUR   REQUESTSINLAST24H
+...
+poddisruptionbudgets.v1.policy                                                               391                     8114
+poddisruptionbudgets.v1beta1.policy                                       1.25               2                       23
+podmonitors.v1.monitoring.coreos.com                                                         3                       70
+podnetworkconnectivitychecks.v1alpha1.controlplane.operator.openshift.io                     612                     11748
+pods.v1                                                                                      1531                    38634
+podsecuritypolicies.v1beta1.policy                                        1.25               3                       39
+podtemplates.v1                                                                              2                       79
+preprovisioningimages.v1alpha1.metal3.io                                                     2                       39
+priorityclasses.v1.scheduling.k8s.io                                                         12                      248
+prioritylevelconfigurations.v1beta1.flowcontrol.apiserver.k8s.io          1.26               3                       86
+...
+----
++
+[IMPORTANT]
+====
+You can safely ignore the following entries that appear in the results:
+
+* The `system:serviceaccount:kube-system:generic-garbage-collector` user might appear in the results because it walks through all registered APIs searching for resources to remove.
+* The `system:kube-controller-manager` and `system:cluster-policy-controller` users might appear in the results because they walk through all resources while enforcing various policies.
```

Comment:
There's a longer list of safe ignores [here][1].  At the moment, those additions all seem to be talking about 4.8-to-4.9, but that information didn't make it back into [the 4.9 release notes][2].  With the guidance split between the docs and the KCS, it's unclear to me which (or both?) are expected to get bumped as new entries are discovered.

[1]: https://access.redhat.com/articles/6955985
[2]: https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-evaluate-apirequestcount_updating-cluster-prepare

Reply:
@wking we are trying to keep them in sync. That #50311 PR is trying to update with new info, and then I'm going to pull that info in here as well, now that it has merged. 

If there is anything outside of those updates, then I'd like to know about it so we can make the updates both places.